src/resources/variable/Variable.yaml

```diff
@@ -1,25 +1,25 @@
 get:
   summary: 'List variables'
-  description: Returns a list of variables
+  description: Returns a list of variables. The result can be filtered by using the query parameters.
   operationId: listVariables
   parameters:
     - in: query
       name: parent_id
       schema:
         type: string
         format: uuid
-      description: the id where the variable will be added
+      description: it filters the list by returning only the variables available to the selected parent_id (parent_id is the id of an application, cronjob or lifecycle job)
     - in: query
       name: scope
       schema:
         $ref: '../../schemas/enums/APIVariableScope.yaml'
-      description: the scope of the parent where the variable will be added
+      description: it filters the list by returning only the variables having the selected scope
```

Comment:
actually, it is the scope of the parent_id.

Reply:
fixed